convert/harness/downgrader/downgrade.go

```diff
@@ -574,6 +579,70 @@ func (d *Downgrader) convertStepPlugin(src *v1.Step) *v0.Step {
 	}
 }
 
+// helper function to convert a Plugin step from the v1
+// structure to the v0 harness structure.
+//
+// TODO convert resources
+// TODO convert reports
```

Comment:
Are these applicable here?

Reply:
Yeah they do exist in the v0 Docker Step